NineChronicles.Headless.AccessControlCenter/AccessControlService/MutableRedisAccessControlService.cs

```diff
@@ -18,10 +18,24 @@ public MutableRedisAccessControlService(string storageUri)
 
         public void DenyAccess(Address address)
         {
-            _db.StringSet(address.ToString(), "denied");
```

Comment:
It is not a service running somewhere, right? If it is running, it should care for backward-compatibility

Reply:
@moreal Thanks for the review!

I've just had a discussion with @Atralupus and we've decided to remove `AllowAccess` and `DenyAccess` completely because the function of the `ACS` has been changed from "controlling access" to "managing tx quotas". I'll make these new updates along with your other suggestion and request one more review. 🙏